src/resources/genomicsdb_export_config.proto

```diff
@@ -98,10 +98,10 @@ message ExportConfiguration {
   optional bool sites_only_query = 23;
   optional bool produce_GT_with_min_PL_value_for_spanning_deletions = 24;
   optional bool scan_full = 25;
-  optional uint32 segment_size = 26;
+  optional uint32 segment_size = 26 [default = 10485760];
   optional uint32 combined_vcf_records_buffer_size_limit = 27;
-  optional bool enable_shared_posixfs_optimizations = 28;
-  optional bool bypass_intersecting_intervals_phase = 29;
+  optional bool enable_shared_posixfs_optimizations = 28 [default = false];
```

Comment:
On a related note, I was thinking we'd want to have `enable_shared_posixfs_optimizations` be enabled by default for performance reasons...or is there some downside? I appreciate that it makes sense to default bools to false in general but...

Reply:
@webberlh, it's nice to have explicit defaults for some booleans, so we know from looking at the protobuf definitions what they are. Otherwise, you may have to look at the underlying code to see what the defaults are. 

@mlathara, the shared posixfs optimizations make only sense for NFS/Lustre, but I get your point that most filesystems are mounted and we need it anyway. Let me think a little more about this.

Reply:
@mlathara, I think we should keep the default to false for shared posixfs optimizations just because that is the default in gatk too. We can clean it up all the way, but I would like to rename this to `disable_shared_posixfs_optimizations` if we want the default true and we should clean up gatk as well. Let me know how you would like this done.

Reply:
Fair enough -- let's keep it the way it is. We can think about the rename but doesn't seem worth it at this point

Reply:
> Is there an advantage to being explicit about the defaults for these two `bool`'s?

Was thinking about @webberlh's question here... Amazingly, no defaults are allowed explicitly in `proto3` [definitions](https://developers.google.com/protocol-buffers/docs/proto3#default). @mlathara, the defaults for booleans in proto3 is false! Probably time to move on to using proto3 and using the default values to initialize explicitly in the GenomicsDB config constructors.

Reply:
@nalinigans thanks for the clarification, that make sense w.r.t. the constructors!

Reply:
@webberlh, should we be using `proto3` for OmicsDS so we can rely on reasonable defaults? Apparently, `proto2` will always be supported as the wire format for both versions is the same. So, it is OK to not switch too. GenomicsDB will require a heavy lift for the migration, so don't recommend for now.

Reply:
@nalinigans I think my only concern would be for losing the ability to check if a field was explicitly set with the default value vs. not set which I think is currently leveraged by the CLI. But it may be that we can work around this. I can create an issue to discuss this further. +1 on leaving GenomicsDB as is.